dpnp/tests/test_utils.py

```diff
@@ -0,0 +1,67 @@
+import dpctl.tensor as dpt
+import numpy
+import pytest
+
+import dpnp
+
+
+class TestIsSupportedArrayOrScalar:
+    @pytest.mark.parametrize(
+        "array",
+        [
+            dpnp.array([1, 2, 3]),
+            dpnp.array(1),
+            dpt.asarray([1, 2, 3]),
+        ],
+    )
+    def test_valid_arrays(self, array):
+        assert dpnp.is_supported_array_or_scalar(array)
+
+    @pytest.mark.parametrize(
+        "value",
+        [
+            42,
+            True,
+            "1",
+        ],
+    )
+    def test_valid_scalars(self, value):
+        assert dpnp.is_supported_array_or_scalar(value)
```

Comment:
Explicit assertion seems more clear about testing scenario.
```suggestion
        assert dpnp.is_supported_array_or_scalar(value) is True
```